core-bundle/public/sectionwizard.js

```diff
@@ -1,118 +1,121 @@
 (function(){
     'use strict';
 
-    const initialized = new WeakMap();
+    const initializedRows = new WeakMap();
 
-    const init = (node) => {
-        node.querySelectorAll('.tl_sectionwizard').forEach((table) => {
-            if (initialized.has(table)) {
-                return;
-            }
+    const init = (row) => {
+        // Check if this row has already been initialized
+        if (initializedRows.has(row)) {
+            return;
+        }
 
-            initialized.set(table, true);
+        // Check if this row has all necessary elements
+        if (7 !== row.querySelectorAll('input, select, button').length) {
```

Comment:
```suggestion
        // Skip header row
        if (row.querySelector('th')) {
```

In my tests, the `row` was either the header…

```
<th>Title</th>
<th>ID</th>
<th>Template</th>
<th>Position</th>
<th></th>
```

…or a valid row with buttons. Could this have been the problem?

Reply:
The `row` variable should always be a `tr` element, since that is what I pass to `init` [here](https://github.com/contao/contao/pull/4939/files#diff-4511fe5a886b411f394444cab142d2d24b5b1726b4a8a1538c7f2b864a3ae4f7R111) and [here](https://github.com/contao/contao/pull/4939/files#diff-4511fe5a886b411f394444cab142d2d24b5b1726b4a8a1538c7f2b864a3ae4f7R118) 🤔

Reply:
Also the check is there in order to disregard any MutationObserver events where not all elements (per `<tr>`) that are required for the functionality of this JavaScript have been parsed yet.